### PR TITLE
Update Date.setTime()

### PR DESCRIPTION
**Pull request by NiedziolkaMichal**
Fix for #2121 
New code output is:
```
> Thu Jul 01 1999 12:00:00 GMT+0200 (CEST)
> Thu Jul 01 1999 12:05:00 GMT+0200 (CEST)
```

### CODE REVIEW

File: live-examples/js-examples/date/date-settime.js [hunk 1]
```diff
@@ -1,10 +1,13 @@
-const event1 = new Date('July 1, 1999');
-const event2 = new Date();
-event2.setTime(event1.getTime());
+const launchDate = new Date('July 1, 1999, 12:00:00');
+const dateInSpace = new Date();
```
Comment (schalkneethling) [inline live-examples/js-examples/date/date-settime.js:2]: Hey @NiedziolkaMichal! Thank you so much for all your wonderful contributions. The variable name `dateInSpace`, do you mean `futureDate` perhaps?

**Review by schalkneethling — COMMENTED** [review]
Just one clarifying question.

File: live-examples/js-examples/date/date-settime.js (hunk 1 above)
Comment (NiedziolkaMichal) [reply live-examples/js-examples/date/date-settime.js]: By `dateInSpace` I ment few minutes after launch of some spaceship, when it passed Kármán line. Variable name doesn't really matter, so I changed it to `futureDate` as you suggested. Thanks

**Review by schalkneethling — APPROVED** [review]
🎉  Thank you for your contribution, @NiedziolkaMichal 🎉

**Pull request merged**